Add more mr.bob, bobtemplates.plone, plonecli references (#1734)
plone/documentation: docs/classic-ui/mockup.md

## Get started

[`bobtemplates.plone`](https://github.com/plone/bobtemplates.plone) provides [`mr.bob`](https://github.com/collective/mr.bob) templates to generate packages for Plone projects.
[Plone CLI (`plonecli`)](https://github.com/plone/plonecli) provides a command line client for `bobtemplates.plone`.

Install {term}`plonecli` into your Python user packages to make it available to all your projects.

```shell
pip install plonecli --user
```

> Create a theme package add-on with `plonecli`.

Reviewer: ```suggestion
Create an add-on package add-on with `plonecli`.
```

Author: Your suggestion has "add-on" twice and confuses me.

Author: I think you meant:

```
Create an add-on package with `plonecli`.
```

Update in https://github.com/plone/documentation/pull/1734/commits/b0881aef23b621f3998a4a54baf71cd2872bac4b

```shell

> plonecli add mockup_pattern

Reviewer: ```suggestion
plonecli create addon project.addon
```

This will create a `project.addon` package which can be installed in your Plone site.

You can `cd` to the project, and add features to that package, such as content-types, behaviors, controlpanels or REST API endpoints.

```shell
cd project.addon
plonecli add content_type
plonecli add behavior
plonecli theme_barceloneta
```

Each of the features asks several questions to create the desired feature as customized as possible.

You can check the full list of available features using the `-l` parameter:

```shell
plonecli -l
```

Author: Added in https://github.com/plone/documentation/pull/1734/commits/b0881aef23b621f3998a4a54baf71cd2872bac4b.

Thanks for the review!
